build/karma.conf.babel.js

```diff
@@ -23,15 +22,18 @@ module.exports = (karmaConfig) => {
       // exit on ResourceError, useful if karma exits without killing phantom
       exitOnResourceError: true,
     },
+    coverageReporter: {
+      reporters: [
+        { type: 'lcov', dir: 'coverage', subdir: '.' },
+        { type: 'text-summary' },
+      ],
+      includeAllSources: true,
+    },
```

Comment:
We report `lcov` data to the `coverage/` folder.  We also show a summary of coverage levels in the console.